cmake/ExternalMFEM.cmake

```diff
@@ -297,6 +301,32 @@ Intel C++ compiler for MUMPS and STRUMPACK dependencies")
       "-DMUMPS_REQUIRED_LIBRARIES=${SCALAPACK_LIBRARIES}$<SEMICOLON>${STRUMPACK_MUMPS_GFORTRAN_LIBRARY}"
     )
   endif()
+
+  # Configure SUNDIALS
+  if(PALACE_WITH_SUNDIALS)
+    set(SUNDIALS_REQUIRED_PACKAGES "LAPACK" "BLAS" "MPI")
+    if(PALACE_WITH_OPENMP)
+      list(APPEND SUNDIALS_REQUIRED_PACKAGES "OpenMP")
+    endif()
+    if(PALACE_WITH_CUDA)
+      list(APPEND SUNDIALS_REQUIRED_PACKAGES "CUDAToolkit")
+      list(APPEND SUNDIALS_REQUIRED_LIBRARIES ${SUPERLU_STRUMPACK_CUDA_LIBRARIES})
+    endif()
+    string(REPLACE ";" "$<SEMICOLON>" SUNDIALS_REQUIRED_PACKAGES "${SUNDIALS_REQUIRED_PACKAGES}")
+    string(REPLACE ";" "$<SEMICOLON>" SUNDIALS_REQUIRED_LIBRARIES "${SUNDIALS_REQUIRED_LIBRARIES}")
+    list(APPEND MFEM_OPTIONS
+      "-DSUNDIALS_DIR=${CMAKE_INSTALL_PREFIX}"
+      "-DSUNDIALS_OPT=-I${CMAKE_INSTALL_PREFIX}/include"
+      "-DSUNDIALS_LIB=-Wl,-rpath,${CMAKE_INSTALL_PREFIX}/lib64 -L${CMAKE_INSTALL_PREFIX}/lib64 -lsundials_arkode, -lsundials_cvode -lsundials_nvecserial -lsundials_kinsol -lsundials_nvecparhyp -lsundials_nvecparallel"
```

Comment:
`nvecserial`? Are we using the serial nvec? Or is this just something that the mfem linking requires?

Reply:
Not needed indeed.